From f16e3bd13a027a0845282f00b40c102dab5936ed Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Thu, 30 Jan 2025 12:29:33 +0000
Subject: [PATCH] Selinux_fc is not used anymore to be defined by Vendor

file_contexts.bin file will be generated by filesystem_creator.go file by
AOSP build. So no need for defining selinux_fc.

Tests Done: Build proceed

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.com>
---
 groups/config-partition/true/AndroidBoard.mk | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/groups/config-partition/true/AndroidBoard.mk b/groups/config-partition/true/AndroidBoard.mk
index fa81cec2..99725e7b 100644
--- a/groups/config-partition/true/AndroidBoard.mk
+++ b/groups/config-partition/true/AndroidBoard.mk
@@ -1,6 +1,5 @@
 INSTALLED_CONFIGIMAGE_TARGET := $(PRODUCT_OUT)/config.img
 
-selinux_fc := $(TARGET_ROOT_OUT)/file_contexts.bin
 
 {{^slot-ab}}
 OEM_CONFIG_DIR := $(TARGET_ROOT_OUT)/oem_config
@@ -30,8 +29,7 @@ $(RECOVERY_OEM_CONFIG_DIR):
 
 $(PRODUCT_OUT)/ramdisk.img: $(OEM_CONFIG_DIR) $(RECOVERY_OEM_CONFIG_DIR)
 
-$(INSTALLED_CONFIGIMAGE_TARGET) : PRIVATE_SELINUX_FC := $(selinux_fc)
-$(INSTALLED_CONFIGIMAGE_TARGET) : $(MKEXTUSERIMG) $(MAKE_EXT4FS) $(E2FSCK) $(selinux_fc) $(INSTALLED_BOOTIMAGE_TARGET) $(OEM_CONFIG_DIR) $(RECOVERY_OEM_CONFIG_DIR)
+$(INSTALLED_CONFIGIMAGE_TARGET) : $(MKEXTUSERIMG) $(MAKE_EXT4FS) $(E2FSCK) $(INSTALLED_BOOTIMAGE_TARGET) $(OEM_CONFIG_DIR) $(RECOVERY_OEM_CONFIG_DIR)
 	$(call pretty,"Target config fs image: $(INSTALLED_CONFIGIMAGE_TARGET)")
 	@mkdir -p $(PRODUCT_OUT)/config
 	$(hide)	PATH=$(HOST_OUT_EXECUTABLES):$$PATH \
@@ -40,9 +38,7 @@ $(INSTALLED_CONFIGIMAGE_TARGET) : $(MKEXTUSERIMG) $(MAKE_EXT4FS) $(E2FSCK) $(sel
 		$(PRODUCT_OUT)/config.img \
 		ext4 \
 		oem_config \
-		$(BOARD_CONFIGIMAGE_PARTITION_SIZE) \
-		$(PRIVATE_SELINUX_FC)
+		$(BOARD_CONFIGIMAGE_PARTITION_SIZE) 
 
 INSTALLED_RADIOIMAGE_TARGET += $(INSTALLED_CONFIGIMAGE_TARGET)
 
-selinux_fc :=
-- 
2.34.1

